dart_conf_flutter/ios/Podfile.lock

```diff
@@ -88,40 +88,40 @@ PODS:
     - Flutter
 
 DEPENDENCIES:
-  - cloud_firestore (from `/Users/devoncarew/.pub-cache/hosted/pub.dartlang.org/cloud_firestore-0.1.0/ios`)
-  - Flutter (from `/Users/devoncarew/projects/flutter/flutter/bin/cache/artifacts/engine/ios`)
-  - url_launcher (from `/Users/devoncarew/.pub-cache/hosted/pub.dartlang.org/url_launcher-1.0.3/ios`)
+  - cloud_firestore (from `/Users/wmleler/.pub-cache/hosted/pub.dartlang.org/cloud_firestore-0.1.2/ios`)
```

Comment:
We may want to delete this podfile file completely, and then put it in the .gitignore - each dev would then have their own local copy.

Reply:
Added the lock file to .gitignore, but did not delete it (yet).

I redid the iOS stuff, and it worked first time this time. Yippee!